### PR TITLE
Social learning

### LINKED ISSUE #6: Social Learning
```
The definition of social learning seems to limit it to humans.  However, there 
is a long history of study of social learning in animals (e.g., Zental and 
Galef 1988).  The definition should be updated to reflect this.
```

Original issue reported on code.google.com by `peter.mi...@gmail.com` on 10 Jun 2011 at 3:22

### PR DESCRIPTION
**Pull request by DitchingIt**
Fixes #6

### CODE REVIEW

File: src/ontology/nbo-edit.owl [hunk 1]
```diff
@@ -3240,10 +3242,13 @@ SubClassOf(obo:NBO_0000219 obo:NBO_0000210)
 
 # Class: obo:NBO_0000220 (social learning)
 
-AnnotationAssertion(obo:IAO_0000115 obo:NBO_0000220 "\"The sociological process of training individuals in a society to act or respond in a manner generally approved by the society in general and peer groups within society.\" [wikipedia:Social_conditioning]")
+AnnotationAssertion(obo:IAO_0000115 obo:NBO_0000220 "associative learning behavior that arises from observing or interacting with another individual")
+AnnotationAssertion(dc:contributor obo:NBO_0000220 <https://orcid.org/ 0000-0002-4277-2151 >)
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:3246]:
```suggestion
AnnotationAssertion(dc:contributor obo:NBO_0000220 <https://orcid.org/ 0000-0002-4277-2151>)
```

File: src/ontology/nbo-edit.owl [hunk 2]
```diff
@@ -3240,10 +3242,13 @@ SubClassOf(obo:NBO_0000219 obo:NBO_0000210)
 
 # Class: obo:NBO_0000220 (social learning)
 
-AnnotationAssertion(obo:IAO_0000115 obo:NBO_0000220 "\"The sociological process of training individuals in a society to act or respond in a manner generally approved by the society in general and peer groups within society.\" [wikipedia:Social_conditioning]")
+AnnotationAssertion(obo:IAO_0000115 obo:NBO_0000220 "associative learning behavior that arises from observing or interacting with another individual")
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:3245]: Source? And capital first letter and trailing period missing.

File: src/ontology/nbo-edit.owl [hunk 3]
```diff
@@ -3240,10 +3242,13 @@ SubClassOf(obo:NBO_0000219 obo:NBO_0000210)
...
+AnnotationAssertion(dc:contributor obo:NBO_0000220 <https://orcid.org/ 0000-0002-4277-2151 >)
 AnnotationAssertion(dc:date obo:NBO_0000220 "2011-04-04T03:34:46Z")
 AnnotationAssertion(oboInOwl:created_by obo:NBO_0000220 "George Gkoutos")
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0000220 "behavior_ontology")
+AnnotationAssertion(oboInOwl:remark obo:NBO_0000220 "Original definition: \"The sociological process of training individuals in a society to act or respond in a manner generally approved by the society in general and peer groups within society.\" [wikipedia:Social_conditioning]")
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:3250]:
```suggestion
AnnotationAssertion(rdfs:comment obo:NBO_0000220 "Original definition: \"The sociological process of training individuals in a society to act or respond in a manner generally approved by the society in general and peer groups within society.\" [wikipedia:Social_conditioning]")
```

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/ontology/nbo-edit.owl (hunk 2 above)
Comment (DitchingIt) [reply src/ontology/nbo-edit.owl]: This is a minor modification to solve a 2015 argument - we are not trying to academically source this definition yet - just point to our reasoning for the solution, but it is on the long list when we get round to reorganising the NBO.

**Review by matentzn — APPROVED** [review]

**Pull request merged**